ui/article-renderer/src/main/java/org/cru/godtools/article/adapter/ArticleAdapter.java

```diff
@@ -0,0 +1,97 @@
+package org.cru.godtools.article.adapter;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import org.cru.godtools.article.R;
+import org.cru.godtools.articles.aem.model.Article;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+import java.util.Locale;
+
+public class ArticleAdapter extends RecyclerView.Adapter<ArticleAdapter.ArticleViewHolder> {
```

Comment:
a lot of this logic disappears when you switch to utilizing databinding. see: https://github.com/CruGlobal/godtools-android/blob/develop/ui/article-renderer/src/main/java/org/cru/godtools/article/adapter/CategoriesAdapter.java